src/ui/components/Collections/Collections.jsx

```diff
@@ -101,9 +101,8 @@ const Collections = props => {
           { value: 'secret', label: 'secret', edit: true },
           // NOTE: secret is only used for client_secret_basic auth.
           // see src/utils/client.js : connectToServer for more info
-          { value: 'customScopes', label: 'customScopes', edit: true },
-          { value: 'type', label: 'type', edit: false }
-        ];
+          { value: 'customScopes', label: 'customScopes', edit: true }
+]        ];
```

Comment:
Extra `]` here
```suggestion
        ];
```

Reply:
i wonder why this didn't produce an error and the linter didn't say anything